_source/logzio_collections/_log-sources/guardduty.md

```diff
@@ -220,13 +220,10 @@ aws cloudformation deploy
 | Parameter | Description |
 |---|---|
 | LogzioTOKEN <span class="required-param"></span> | {% include log-shipping/replace-vars.html token=true %} <!-- logzio-inject:account-token --> |
-| KinesisStream <span class="required-param"></span> | The name of the Kinesis stream where this function will listen for updates. |
-| LogzioURL <span class="default-param">`https://listener.logz.io:8071`</span> | {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
-| LogzioTYPE <span class="default-param">`logzio_kinesis_stream`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
+| LogzioURL <span class="default-param">`https://listener.logz.io:8071`</span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
+| LogzioTYPE <span class="default-param">`logzio_cloudwatch_logs`</span> | The log type you'll use with this Lambda. This can be a [built-in log type]({{site.baseurl}}/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You should create a new Lambda for each log type you use. |
```

Comment:
Guardduty takes the logs from kinesis so the kinesis params should be here. You can understand it from step 2